eks-without-vpc/variables.tf

```diff
@@ -0,0 +1,82 @@
+#
+# Variables Configuration
+#
+
+variable "cluster-name" {
+  type    = "string"
+}
+
+variable "ec2-instance-type" {
+  type    = "string"
+}
+
+variable "max-ec2-instances" {
+  default = 1 
```

Comment:
Max and desired default values should be increased to 3.

Reply:
increased